src/main/kotlin/lotto/controller/LottoApp.kt

```diff
@@ -14,8 +14,14 @@ class LottoApp {
 
     fun startLotto() {
         val count = inputView.inputUser()
-        val tickets = GeneratorLottoNumbers.generateRandomLottoTickets(count)
-        resultView.printBuyTickets(tickets)
+        val manualTickets = inputView.inputManualTickets()
+        val autoTicketCount = count - manualTickets.size
+        val tickets = GeneratorLottoNumbers.generateRandomLottoTickets(autoTicketCount)
```

Comment:
```suggestion
        val manualTickets = inputView.inputManualTickets()
        val autoTicketCount = count - manualTickets.size
        val autoTickets = GeneratorLottoNumbers.generateRandomLottoTickets(autoTicketCount)
```

tickets보다 autoTickets이 더 알맞는 네이밍으로 보이네요~